hw1/02-palindrome.js

```diff
@@ -1,3 +1,23 @@
 const elem = document.querySelector('input');
-
+const resultText = document.getElementById('resultText');
 elem.addEventListener('input', handleInput);
+
+function handleInput(){
```

Comment:
I think you might want to use the `event` parameter passed to the on-change handle instead so this function is more reusable in case we have more `input`